tests/test_tasks/test_task.py

```diff
@@ -65,10 +119,32 @@ def _upload_dataset(self, dataset: OpenMLDataset) -> int:
             dataset.
         """
         dataset.name = '%s%s' % (self._get_sentinel(), dataset.name)
-        try:
-            return dataset.publish()
-        except openml.exceptions.OpenMLServerException:
-            # Something went wrong.
-            # Test dataset was not
-            # published. Return old id.
-            return dataset.dataset_id
+        # Providing both dataset file and url
+        # raises an error when uploading.
+        dataset.url = None
+
+        return dataset.publish()
+
+    @staticmethod
+    def _wait_dataset_activation(
```

Comment:
Are you still using this method? I couldn't find any usage of it.

Reply:
Regarding this function and the one below, I am not using them. I created them with the initial implementation of task upload that we ended up not using. I left the functions since they might be of use in the future.
However, I can comment them out, or remove them.

Reply:
Yes, please remove them then.